packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -242,7 +249,7 @@ function AuthOptions({
   };
 
   useEventListener(globalThis, 'message', async (e) => {
-    if (e.data?.eventKey !== AuthEvent.SocialRegistration) {
+    if (e.data?.eventKey !== AuthEvent.SocialRegistration || ignoreMessages) {
```

Comment:
Because of the ignore here don't you miss below set error?

Reply:
Yes, we won't since the modal is opened, it has its own listener (with `ignoreMessages = false`) which allows the whole process to be completed.

Reply:
Got it :)